project.clj

```diff
@@ -5,7 +5,8 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [clj-aws-s3 "0.3.10"]
+                 ;; avoid warning about version ranges
+                 [clj-aws-s3 "0.3.10" :exclusions [commons-codec joda-time]]
+                 [joda-time "2.2"]
                  [org.clojure/core.async "0.2.395"]
-                 [fivetonine/collage "0.2.1"]])
-                 
\ No newline at end of file
+                 [fivetonine/collage "0.2.1" ]])
```

Comment:
This stuff isn't really a refactoring, just some ambiguous version ranges that I wanted to fix. Basically I just locked to a specific version of joda-time so are pinned to a specific version instead of a range. No need to worry about this stuff right now.